src/api/spec/controllers/webui/configuration_controller_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'rails_helper'
+
+RSpec.describe Webui::ConfigurationController do
+  let(:confirmed_user) { create(:confirmed_user) }
+  let(:admin_user) { create(:admin_user) }
+
+  describe 'GET #index' do
+    it { should use_before_action(:require_admin) }
+
+    it 'is allowed as admin User' do
```

Comment:
I do trust it but I also want to test the functionality of the actions (and this is only possible as admin). We can discuss about renaming the test, so the name isn't focused on the admin privileges.